bluebrain/deployment/environments/applications_hpc.yaml

```
@@ -51,7 +51,7 @@ spack:
     - neurodamus-thalamus+coreneuron+caliper%intel^coreneuron+knl
     - neuron%intel+coreneuron+tests
     - neuron%nvhpc+coreneuron+tests ^coreneuron+gpu~shared
-    - neuron%intel+debug
+    - neuron%intel+coreneuron+debug
```

Maybe this needs to be
```suggestion
    - neuron%intel+coreneuron+debug build_type=Debug
```
to match https://bbpgitlab.epfl.ch/hpc/sim/blueconfigs/-/blob/main/.tests_setup.sh#L23?